schema/artist/index.js

```diff
@@ -157,9 +169,29 @@ const ArtistType = new GraphQLObjectType({
               type: GraphQLString,
               resolve: ({ credit }) => credit,
             },
+            partner_id: {
+              type: GraphQLString,
+              resolve: ({ partner_id }) => partner_id,
+              description: 'The partner id of the partner who submitted the featured bio.',
+            },
           },
         }),
-        resolve: ({ blurb, id }, { format }) => {
+        resolve: ({ blurb, id }, { format, partner_bio }) => {
+          if (partner_bio) {
+            return gravity(`artist/${id}/partner_artists`, {
+              size: 1,
+              featured: true,
+            }).then((partner_artists) => {
+              if (partner_artists && partner_artists.length) {
+                const { biography, partner } = first(partner_artists);
+                return {
+                  text: biography,
+                  credit: `Submitted by ${partner.name}`,
+                  partner_id: partner.id,
+                };
+              }
+            });
```

Comment:
So this is the case of passing in `partner_bio: true`.

The current behavior is to always return the Artsy bio, otherwise a featured bio. However, since we un-deprecate `blurb` to return the Artsy bio, we can use that param to just return the featured bio here. I think that makes sense b/c it leaves the existing behavior/clients as-is, plus this field is already designed to return a partner-submitted bio (with credit line).